runtime/manta-pc/src/lib.rs

```diff
@@ -595,6 +660,33 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = pallet_collator_selection::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	pub const MantaXassetsPalletId: PalletId = PalletId(*b"/ma_xast");
+
+	pub const AnyNetwork: NetworkId = NetworkId::Any;
+	pub FriendChains: Vec<(MultiLocation, u128)> = vec![
+		// Acala local and live, 0.01 ACA
```

Comment:
What does this comment mean ?

Reply:
This is a tuple, the first element is which parachain we support, it likes a white list.
The second one is the min token to transfer while making cross chain transaction.

Reply:
Maybe just add it as a comment.

Reply:
Sure